organizations/admin.py

```diff
@@ -27,6 +27,9 @@ def get_actions(self, request):
 
         return actions
 
+    @admin.action(
```

Comment:
Please link the release notes for this change.

Reply:
https://github.com/adamchainz/django-upgrade